lib/web/command_utils.go

```diff
@@ -0,0 +1,100 @@
+/*
+
+ Copyright 2023 Gravitational, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+
```

Comment:
What's with all these random extra lines in the license headers? Did you use `make fix-license` to generate these?

Reply:
Bad IDE template. Fixed already, but my fix only works for new files, not already created ones.